kpatch-build/kpatch-elf.c

```diff
@@ -65,6 +65,18 @@ bool is_text_section(struct section *sec)
 		(sec->sh.sh_flags & SHF_EXECINSTR));
 }
 
+bool is_patchable_function_entries_section(struct section *sec)
+{
+	char *name;
+
+	if (is_rela_section(sec))
+		name = sec->base->name;
+	else
+		name = sec->name;
+
+	return !strncmp(name, "__patchable_function_entries", 28);
```

Comment:
The `__patchable_function_entries` has 28 chars without the `\0`. So if the name is `__patchable_function_entries_abcd`, this function will also return a true. 
Is this expected?

Reply:
If we believe that would never happen, why not `strncmp (.., ..., 29)` to include the \0 ? This can make this code more robust.
Or just as Song suggested, with `strcmp`.